easybuild/easyconfigs/g/GDAL/GDAL-3.11.3-intel-2025b.eb

```
@@ -0,0 +1,79 @@
+easyblock = 'CMakeMake'
+
+name = 'GDAL'
+version = '3.11.3'
+
+homepage = 'https://www.gdal.org'
+description = """GDAL is a translator library for raster geospatial data formats that is released under an X/MIT style
+ Open Source license by the Open Source Geospatial Foundation. As a library, it presents a single abstract data model
+ to the calling application for all supported formats. It also comes with a variety of useful commandline utilities for
+ data translation and processing."""
+
+toolchain = {'name': 'intel', 'version': '2025b'}
+toolchainopts = {'usempi': True, 'pic': True, 'optarch': '-march=x86-64-v3'}
```

Hardcoding `-march=x86-64-v3` here is not a good idea.

Maybe try using this instead:
```suggestion
toolchainopts = {'usempi': True, 'pic': True, 'precise': True}
```

That will avoid the use of `-fp-speculation=safe`, so it's a way of dancing around the problem that @Thyre mentioned